Tests/NIOHTTPClientTests/SwiftNIOHTTPTests.swift

```diff
@@ -170,6 +170,32 @@ class SwiftHTTPTests: XCTestCase {
         response = try httpClient.get(url: "http://localhost:\(httpBin.port)/redirect/https?port=\(httpsBin.port)").wait()
         XCTAssertEqual(response.status, .ok)
     }
+    
+    func testHttpHostRedirect() throws {
+        let httpClient = HTTPClient(eventLoopGroupProvider: .createNew,
+                                    configuration: HTTPClient.Configuration(certificateVerification: .none, followRedirects: true))
+        
+        defer {
+            try! httpClient.syncShutdown()
+        }
+        
+        let response = try httpClient.get(url: "https://httpbin.org/redirect-to?url=https%3A%2F%2Fwww.httpbin.org%2Fheaders").wait()
```

Comment:
What about using `127.0.0.1` as a target for redirect in the local `HttpBin` implementation?

Reply:
@artemredkin Thank you for this tip, done in latest commit! :)